doc/README.md

```diff
@@ -1,3 +1,19 @@
+## Deprecation Notice
+
+The new open source [Citus extension](https://github.com/citusdata/citus)
+for PostgreSQL provides a superset of the functionality of `pg_shard`.
```

Comment:
Minor.

``for PostgreSQL provides a superset`` --> ``for PostgreSQL natively integrates pg_shard and provides a superset of its functionality``, or a better wording of this to signal that Citus integrates pg_shard in full?